FriishProduce/Strings/es.json

```diff
@@ -203,8 +205,8 @@
 
 			"use_online_wad_enabled":           "Habilitar opción de descargar canales de origen",
 			"use_online_wad_tip":				"Esto se puede utilizar para descargar cualquier canal de origen que esté disponible en la base de datos actual sin necesidad de tener un archivo WAD existente.
-												Sin embargo, debido a que el cliente NUS de libWiiSharp no permite descifrar archivos WADs de la Consola Virtual de ninguna forma, solo es posible utilizar la función de descarga desde la Web.
-												La legalidad de esta función es dudosa, por lo que es mejor tener esta opción deshabilitada.",
+												\nSin embargo, debido a que el cliente NUS de libWiiSharp no permite descifrar archivos WADs de la Consola Virtual de ninguna forma, solo es posible utilizar la función de descarga desde la Web.
```

Comment:
I have added in these lines the **\n** string for add a space between paragraphs.